### PR TITLE
Fix license headers

### PR DESCRIPTION
**Pull request by PierreRambaud**
This change is [Reviewable](https://reviewable.io/reviews/prestashop/ps_facetedsearch/175)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,46 +1,49 @@
...
-  "require-dev": {
-    "friendsofphp/php-cs-fixer": "^2.14",
-    "phpunit/phpunit": "~5.7",
-    "prestashop/php-coding-standards": "dev-master",
-    "mockery/mockery": "^1.2"
-  },
-  "config": {
-    "platform": {
-      "php": "5.6.0"
+    "name": "prestashop/ps_facetedsearch",
```
Comment (matks) [inline composer.json:2]: Indented but I think it's harmess

Comment (PierreRambaud) [reply composer.json:2]: I even don't know how the indentation was changed (I only used header-stamp :facepalm: )

Comment (matks) [reply composer.json:2]: It's a known bug of header-stamp. It decodes the entire JSON, adds its stamp, then use `file_put_contents(json-encode($content));` to put content back into composer.json in so the indent used will be the one used on the machine (dont know how it's chosen).

Comment (PierreRambaud) [reply composer.json:2]: :facepalm:

**Pull request merged**